content/docs/learning-guide/tutorials/entities-and-prefabs/override-a-prefab.md

```diff
@@ -40,14 +40,15 @@ Overrides are not limited to the level. In fact, any prefab that is open for edi
 ## Enable prefab overrides
 
 To enable prefab overrides in Entity Outliner, you must set the `ed_enableOutlinerOverrideManagement` console variable (CVar) to "true". Although you *can* set this in the **Console Variables Editor**, this feature is not currently changeable
-while the Editor is running, so it must be set before the Editor starts, via a settings registry file. You should create a settings registry file with the following contents:
+while the Editor is running, so it must be set before the Editor starts, via a settings registry file. You will also want to enable the `ed_enableInspectorOverrideManagement` flag, so that you can edit overrides in the Inspector. You should create a settings registry file with the following contents:
```

Comment:
Since the DPE and prefab overrides are being turned on by default, don't we have to update this section to mention how to disable it instead?

Reply:
yes, that will be in a separate PR for the point release. This is just for the existing documentation